de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -335,9 +340,12 @@ public void onFailure(Integer statusCode, Throwable caught) {
 
             @Override
             public void onSuccess(Void result) {
-                app.setFavorite(!app.isFavorite());
-                eventBus.fireEvent(new AppFavoritedEvent(app));
                 eventBus.fireEvent(new AppUpdatedEvent(app));
+                if (!app.isFavorite()) {
+                    announcer.schedule(new SuccessAnnouncementConfig(appearance.favSuccess(app.getName())));
+                } else {
+                    announcer.schedule(new SuccessAnnouncementConfig(appearance.unFavSuccess(app.getName())));
+                }
```

Comment:
There was no feedback to user when an app was marked or removed as favorites. Added user feedback.